.gitignore

```diff
@@ -21,7 +21,11 @@ mock_*
 
 # Misc
 .DS_Store
+ignore/
+
+# Goreleaser
+dist/
 
 # Optional workspace
 go.work
-*.code-workspace
\ No newline at end of file
+*.code-workspace
```

Comment:
please don't introduce whiteline changes, use `git add -p`

Reply:
will do but this one won't go away

Reply:
fixed in https://github.com/G-Research/fasttrackml/pull/140/commits/3634b46ef265a3f2441f21eddb36d3453d995626